packages/web-app/src/components/common/AddFileForm/FilesList.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import {
+  IconButton,
+  ListSubheader,
+  List,
+  ListItem,
+  InputAdornment,
+  Typography
+} from '@material-ui/core';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import DeleteIcon from '@material-ui/icons/Delete';
+import UndoIcon from '@material-ui/icons/Undo';
+import StringInput from '../Form/StringInput';
+import { IS_DELETED } from './FileHelpers';
+
+const FilesList = ({ files, updateFileName, removeFile, undoRemove }) => {
+  const { formatMessage } = useIntl();
+
+  const isFileDeleted = file => {
+    return file.state === IS_DELETED;
+  };
+
+  return (
+    <List
+      disablePadding
+      subheader={
+        <ListSubheader disableGutters>
+          {formatMessage({ id: 'Files list' })}
+        </ListSubheader>
+      }>
+      {files.map((file, index) => {
+        const isDeleted = isFileDeleted(file);
+        return (
+          // eslint-disable-next-line react/no-array-index-key
+          <ListItem key={index} style={isDeleted ? { opacity: 0.4 } : {}}>
```

Comment:
```suggestion
      {files.map((file, index) =>(
          <ListItem key={file.name} style={isFileDeleted(file) ? { opacity: 0.4 } : {}}>
```
 // key = file.name || file.id ?

Reply:
file id n'est pas forcément défini (lorsque l'utilisateur import un fichier il n'a pas encore été créé dans la bdd) et 2 fichiers peuvent avoir le même nom. Je ne sais pas ce que j'avais mis à l'origine mais c'était la seule solution qui marchait correctement pour moi (parce que du coup quand 2 items avaient la même clé ça faisait tout buggé)